### ISSUE TITLE
[basic-apps] App icons are broken in master

### ISSUE THREAD

**Issue by maximbaz** [KELiON/cerebro#256]
Look at the small app icons on the left side.

Before:
![image](https://cloud.githubusercontent.com/assets/1177900/24146211/fc9dee8c-0e34-11e7-990f-24f35bba06aa.png)

After:
![image](https://cloud.githubusercontent.com/assets/1177900/24146190/e4836a66-0e34-11e7-9619-f3144249e6ac.png)

I've bisected it to the following commit: https://github.com/KELiON/cerebro/commit/4c96a1c7f70a7b52e650585f622baac698d9ae3c

@SkaceKamen could you please have a look? I'm not sure what you were trying to achieve with that commit to make my own fix.

**Comment by maximbaz** [comment]
Looks like, before the https://github.com/KELiON/cerebro/commit/4c96a1c7f70a7b52e650585f622baac698d9ae3c, the icon variable was pointing to an image file, and after that commit the icon is pointing to a desktop file:

```
  {
    "name": "Google Chrome",
    "description": "Access the Internet",
    "exec": "/usr/bin/google-chrome-stable %U",
    "hidden": false,
    "icon": "/usr/share/icons/hicolor/256x256/apps/google-chrome.png",  <-------
    "filename": "google-chrome.desktop",
    "id": "google-chrome.desktop",
    "path": "/usr/share/applications/google-chrome.desktop" <------
  },
```

**Comment by KELiON** [comment]
I'd say moving `icon: path` from `basic-apps/index` to `basic-apps/windows` should fix this problem

**Comment by obriat** [comment]
On Windows both icons are missing
![cerebro 2017-03-24 10 29 49](https://cloud.githubusercontent.com/assets/26036776/24288237/de78e624-107c-11e7-948d-a26a9653dfe9.png)

**Comment by maximbaz** [comment]
Fixed with #265.
